conda/common/configuration.py

```diff
@@ -1600,3 +1601,56 @@ def typify_parameter(self, parameter_name, value, source):
 
     def get_descriptions(self):
         raise NotImplementedError()
+
+
+def unique_sequence_map(
+    *, unique_key: str, property_name: str, allowed_keys: set | None = None
```

Comment:
`property_name` is unused

```suggestion
    *, unique_key: str, allowed_keys: set | None = None
```